app/utils/JSONHelper.ts

```diff
@@ -1,23 +1,35 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
 const types = {
     BigInt: 'bigint',
+    Date: 'date',
 };
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function replacer(this: any, k: string, v: any) {
+    if (typeof v === types.BigInt) {
+        return { '@type': types.BigInt, value: v.toString() };
+    }
+    if (this[k] instanceof Date) {
+        return { '@type': types.Date, value: v };
```

Comment:
Why do we need special handling of Dates now?

Reply:
We need the special handling of Dates, because AccountInfo now contain dates (instead of just strings).
We don't have dates in the "wallet" export, and when exporting a scheduled transfer, the date is a timestamp string.